### DISCUSSION TITLE
2024 Azure Developers JavaScript Days

### DISCUSSION THREAD

**Discussion by juliamuiruri4** [Azure-Samples/contoso-real-estate#447] [General]
# Azure Developers JavaScript Days 💻March 6 & 7, 2024!

![JS-Day](https://github.com/Azure-Samples/contoso-real-estate/assets/40116776/07cc7248-141f-4ac3-a039-dc5b61c5540b)

<p align="center">
  <a href="https://aka.ms/AzureDevelopers/JavaScriptDay"><img src="https://img.shields.io/badge/Event Registration-green?style=for-the-badge" alt="Event Registration Join the Cloud Skills Challenge" border="0" /></a>
  <a href="https://aka.ms/AzureJSDay-Challenge"><img src="https://img.shields.io/badge/Join the Cloud Skills Challenge-gray?style=for-the-badge" alt="Join the Cloud Skills Challenge" border="0" /></a>
</p>

Welcome to the Azure Developers JavaScript Days Discussion Forum 💬 and build your network with other JavaScript developers across the world and Microsoft Experts.

## Your next steps

- Introduce yourself👋 on this forum and start networking 💬
- Join our [Cloud Skills Challenge](https://aka.ms/AzureJSDay-Challenge) 📖to help you polish up on your JavaScript skills
- Fork & Star ⭐ the [Contoso Repo](https://github.com/Azure-Samples/contoso-real-estate). Walk-through the repo to familiarize yourself with the project & codebase. Should you have any questions, feel free to post them here and someone will help you out!

## Prerequisites

1. **Visual Studio Code**. Free to [download](https://code.visualstudio.com/Download)
1. **GitHub Codespaces**. Free [quota](https://github.com/features/codespaces)
1. **GitHub Copilot**. Optional
1. **Microsoft Azure**. Sign up for a free Azure Account [here](https://azure.microsoft.com/free/?WT.mc_id=javascript-120210-juliamuiruri). Students, activate your [free Azure for Student](https://aka.ms/Azurestudentbenefit) benefit

## Day 1 Session Outline 

1. GitHub Copilot can Do That? 😲
2. Building a Versatile RAG Pattern chatbot with Azure OpenAI, LangChain 🔗
3. LangChain.js + Azure: A Generative AI App Journey 🛣️
4. Extending Copilot for Microsoft 365 using JavaScript and TypeScript 💻
5. Have a safe coffee chat with your documentation using Azure AI Services ☕

**Comment by sinedied** [comment]
👋  Feel free to ask questions here about the talks, demos, cloud skills challenge or anything you feel might be useful

**Reply by anfibiacreativa** [reply]
Hi @sinedied Folks joining your session yesterday were asking about your setup during your demo: OS, computer specs, etc. Can you tell us a bit more about it?

**Comment by Nehas129** [comment]
For those who have visited or are familiar with **[M3M Jewels](https://www.m3mjewels.in/)**, what can customers and clients expect in terms of experience and amenities? Share your observations and experiences.
